db/holiday.sql

```diff
@@ -0,0 +1,27 @@
+--- DATABASE FOR HOLIDAY ME ---
+
+DROP TABLE IF EXISTS requests cascade;
+DROP TABLE IF EXISTS request_status cascade;
+
+--- REQUEST STATUS ---
+
+CREATE TABLE request_status(
+    request_status_id serial primary key NOT NULL,
+    req_status varchar(10)
+);
+
+INSERT INTO request_status(req_status) VALUES ('PENDING');
```

Comment:
I think you should  explicitly insert request_status_id here, just in case someone comes along later and reorders these insertions :)